Hyperbolic_triangulation_2/demo/Hyperbolic_triangulation_2/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Created by the script cgal_create_cmake_script
+# This is the CMake script for compiling a CGAL application.
+
+project (Hyperbolic_triangulation_2_Demo)
+
+# Find includes in corresponding build directories
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Instruct CMake to run moc automatically when needed.
+set(CMAKE_AUTOMOC ON) 
+
+cmake_minimum_required(VERSION 2.8.11)
+if(POLICY CMP0043)
```

Comment:
This should be 
if(NOT POLICY CMP0070 AND POLICY CMP0053)

to silence the warning, because after CMake 3.10 there is a deprecation warning.

Reply:
Actually, there is really no reason now to set `CMP0043`, and the minimal CMake version must be 3.1.

Reply:
same for the other demo.

Reply:
@maxGimeno  can you please just fix it.